taxi/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+from taxi_service import settings
```

Comment:
According to the checklist, you should use `AUTH_USER_MODEL` from `django.conf` instead of importing `settings` from your project. This is considered a best practice.